assignments/lambda-classes.js

```diff
@@ -1 +1,90 @@
 // CODE here for your Lambda Classes
+class Person {
+  constructor (attributes){
+    this.name = attributes.name;
+    this.age = attributes.age;
+    this.location = attributes.location;
+  }
+  speak(){
+    return `Hello my name is ${this.name}, I am from ${this.location}`;
+  }
+}
+
+class Instructor extends Person{
+  constructor(InstructorAttrabutes){
+    super(InstructorAttrabutes);
+    this.specialty = InstructorAttrabutes.specialty;
+    this.favLanguage= InstructorAttrabutes.favLanguage;
+    this.catchPhrase = InstructorAttrabutes.catchPhrase;
+  }
+  demo(subject){
+    console.log(`Today we are learning about ${subject}`);
+  }
+  grade(student, name){
```

Comment:
grade receives a student object and a subject string as arguments and logs out '{student.name} receives a perfect score on {subject}'

You used student and name.  Parameter names don't matter as long as you're using them correctly, but you're using `this.student`.  `this.student` represents a student property in the `Person`'s object body.  You need `student.name`